### ISSUE TITLE
Segmentation fault error

### ISSUE THREAD

**Issue by sweetdreams035** [pocketnetteam/pocketnet.core#153]
I am running a pocketcoin node and helping my friend run one as well. He has Ubuntu and I have Windows. I have updated the node to the current version, have the latest database archive and made sure all the folders are not set to read only and I am getting the error "segmentation fault (core dumped)" when trying to start the node on both computers. Does anyone have any suggestions?

**Comment by tawmaz** [comment]
It is missing from the notes on the instructions, but you need to copy the file main.sqlite3 in the release to location ~/.pocketcoin/checkpoints/main.sqlite3 on Linux or on Windows to the location $HOME/appdata/roaming/pocketcoin/checkpoints/main.sqlite3
We really need to figure out a better way to package this.  Also please be sure you are running the 0.20.17 release which just came out a few hours ago.  0.20.16 has a known segmentation fault.
Please also post the contents of your pocketcoin.conf file.

**Comment by sweetdreams035** [comment]
Thank you for the quick response. I am running 0.20.17 and have copied the main.sqlite3 file to the checkpoints folder and it is still crashing on my windows machine. I will keep trying. I will try it later on my friends linux machine and see what the results are.

**Comment by tawmaz** [comment]
Can you post debug.log from the pocketcoin folder?  Please delete it first, then restart the pocketcoin software so that we get only the latest run, otherwise it will append it to the previous file.
Also are you running pocketcoind (terminal) or pocketcoin-qt (gui)?  Try both in case this is an issue with the gui.

**Comment by sweetdreams035** [comment]
Ok,attaching the debug.log file. I have tried running from the command line as well as the GUI, same result
[debug.log](https://github.com/pocketnetteam/pocketnet.core/files/7849862/debug.log)
.

**Comment by sweetdreams035** [comment]
I see the config file is missing?

**Comment by sweetdreams035** [comment]
Here is the debug.log from my friends linux computer
[debug.log](https://github.com/pocketnetteam/pocketnet.core/files/7850655/debug.log)
Also,opening his pocketcoin.config file, it is empty

**Comment by tawmaz** [comment]
We should be able to operate without the pocketcoin.conf file without issue, but possibly there could be some issue.  Here is the contents of the pocketcoin.conf file we have recommended for recent releases:
```
# Enable public API interfaces
api=1
wsuse=1
daemon=1

# Setting ports
publicrpcport=38081
wsport=8087

# Setting work queue and threads
rpcpublicworkqueue=3000
rpcpublicthreads=30
rpcpostworkqueue=1500
rpcpostthreads=15

maxconnections=16
```

Can you share with us the system config of the Linux system including OS distribution, memory size, CPU type, disk speed and size, network connection type, etc?

**Comment by tawmaz** [comment]
Closing this issue as we had a segmentation fault fix in 0.20.18 which occured when pocketcoin-qt was run with an empty or missing pocketcoin-qt file.
